index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Asset Management</title>
+    <script src="https://kit.fontawesome.com/ce489bc768.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+            <div class="listItems">
+                <p>Today's To Do</p>
+                <div class="refresh"><i class="fa-solid fa-rotate fa-sm"></i></div>
+            </div>
+            <form class="listItems">
+                <input type="text" id="starter" placeholder="Add to your list...">
```

Comment:
Your codes look very nice and well-arranged. I think it will be more appropriate if you change the input` id='starter'` to something more meaning full to understand what this id is indicating to what or which. Please take a look if you feel so as follows : [HTML, and CSS Best Practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
id changed from 'starter' to 'user-input'